bindsnet/network/__init__.py

```diff
@@ -231,6 +231,8 @@ def run(self, inpts: Dict[str, torch.Tensor], time: int, **kwargs) -> None:
         :param Union[float, torch.Tensor] reward: Scalar value used in reward-modulated learning.
         :param Dict[Tuple[str], torch.Tensor] masks: Mapping of connection names to boolean masks determining which
                                                      weights to clamp to zero.
+        :param Union[int, Dict[str, int]] input_time_dim: Dimension for slicing in time for inputs. Can vary 
```

Comment:
Is this necessary? Can it just always be 0?

Reply:
This was the simplest way I could think of to deal with the consequences of the pytorch DataLoader and default_collate_fn which will make everything BxTxCxWxH. Essentially we would have to maintain our own collate_fn and notation for which tensors should have the batch as the first dimension and which shouldn't.